taca/illumina/Runs.py

```diff
@@ -703,6 +703,31 @@ def _process_simple_lane_with_single_demux(
                 os.unlink(dest)
         os.symlink(source, dest)
 
+        # Replace the file laneBarcode.html
+        source_html_report_laneBarcode = os.path.join(
+            source, "html", self.flowcell_id, "all", "all", "all", "laneBarcode.html"
+        )
+        dest_html_report_laneBarcode = os.path.join(
+            dest, "html", self.flowcell_id, "all", "all", "all", "laneBarcode.html"
+        )
+        if os.path.exists(source_html_report_laneBarcode) and os.path.exists(
+            dest_html_report_laneBarcode
+        ):
+            html_report_laneBarcode_parser = LaneBarcodeParser(
+                source_html_report_laneBarcode
+            )
+            # Remove the trailing "_SX" postfix from samples names for BCL Convert when it handles SmartSeq3 libraries
+            for entry in html_report_laneBarcode_parser.sample_data:
+                if "_S" in entry["Sample"]:
```

Comment:
`_S` here but `_SX` in the comment. Any reason for having just `_S` in the code?

Reply:
Hi, `X` is a number such as _S1, S15

Reply:
Ah, yes. That makes sense.